Source/Details/_ASDisplayLayer.mm

```diff
@@ -209,7 +169,7 @@ - (void)display
   ASDisplayNodeAssertMainThread();
   [self _hackResetNeedsDisplay];
```

Comment:
It's time to rename this method. YYAsyncLayer does the same thing :). Maybe just _clearNeedsDisplayFlag